ysh/val_ops.py

```diff
@@ -406,7 +406,7 @@ def ExactlyEqual(left, right):
 
             return True
 
-    raise NotImplementedError(left)
+    raise error.TypeErr(left, "Cannot compare this type", loc.Missing)
```

Comment:
Can we add an optional `loc_t` param  to ExactlyEqual(), and then pass `blame_loc=op` from `ysh/expr_eval.py` where it's called?

I think this will blame the `===` which is nicer